### ISSUE TITLE
Staging Review finding: Include `aria-modal` attribute for sign in modal

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94526]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Identity
**Product:** Sign in page (and sign in modal)
**Feature:** Sign in Transition

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

The sign in modal is structured a little differently than the [`va-modal` component](https://design.va.gov/components/modal/). For the most part I think those differences make sense, but one easy improvement would be to include the `aria-modal="true"` attribute on the `#signin-signup-modal` element.

`aria-modal` helps communicate information about the state of the page to screen reader users. It's not strictly necessary to complete the interaction, but it helps to clarify the nature of the interaction to users who don't get any visual cues to indicate the modal behavior.

## Recommended action

Add `aria-modal="true"` to the `#signin-signup-modal` element.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by claytonzook** [comment]
Not doing this work before launch. For some add'l context, see [this slack thread](https://dsva.slack.com/archives/C068QM7CCKG/p1729009788567159?thread_ts=1727907773.271199&cid=C068QM7CCKG)
